packages/app-headless-cms/src/admin/views/EnvironmentAliases/environmentAliasesUtils.ts

```diff
@@ -15,13 +16,46 @@ export const updateCacheAfterCreate = (cache, updated) => {
         variables: { sort: { savedOn: -1 } }
     };
 
+    const environmentListAliasParams = {
+        query: LIST_ENVIRONMENT_ALIASES
+    };
+
     let data: any;
+    let aliases: any;
+
+    try {
+        aliases = cloneDeep(cache.readQuery(environmentListAliasParams));
+
+        const slug = updatedData.slug;
+        aliases.cms.environmentAliases.data = [
+            ...aliases.cms.environmentAliases.data,
+            {
+                ...updatedData,
+                createdOn: Date.now(),
+                url: {
+                    manage: `/cms/manage/${slug}`,
+                    preview: `/cms/preview/${slug}`,
+                    read: `/cms/read/${slug}`,
+                    __typename: "CmsEnvironmentAliasUrl"
+                }
+            }
+        ];
+
+        cache.writeQuery({
+            ...environmentListAliasParams,
+            data: aliases
+        });
+    } catch (err) {
+        console.log(err);
```

Comment:
Just a short question, can you explain this `try` / `catch`? The `writeQuery` / `readQuery` functions throw errors in specific cases?

Reply:
I saw the `try/catch` for cloning the parameters of the `gqlParams`, so I figured it might be a safe bet to wrap the code in a similar `try/catch` in case the `writeQuery` format no longer matches the right format for `LIST_ENVIRONMENT_ALIASES`. Though we probably don't need console.log there for sure

Reply:
I think we can remove `try` / `catch`. Not sure why we'd want to swallow the error if it happens.

Ofc, correct me if I'm missing something.